cirq-core/cirq/experiments/xeb_fitting_test.py

```diff
@@ -395,3 +396,28 @@ def test_options_defaults_set():
         phi_default=0.0,
     )
     assert o3.defaults_set() is True
+
+
+@pytest.mark.parametrize(
+    'gate',
+    [
+        cirq.CZ,
+        cirq.SQRT_ISWAP,
+        cirq.SQRT_ISWAP_INV,
+        cirq.ISWAP,
+        cirq.ISWAP_INV,
+        cirq.cphase(0.1),
+        cirq.CZ**0.2,
+    ]
+    + [cirq.PhasedFSimGate(*r) for r in (np.pi * np.random.default_rng(0).random((10, 5)))],
```

Comment:
for safety, can we make this random gate with angle from -pi to pi instead of 0 to pi?

Reply:
done